spring-auto-restdocs-docs/snippets.adoc

```diff
@@ -296,6 +296,151 @@ Configuration available during MockMvc setup:
 
 include::section-snippet.adoc[]
 
+[[snippets-links]]
+=== Links snippet
+
+link:{core-package}/hypermedia/LinksSnippet.java[Links snippet] lists all possible links inside the response.
+The links usually cannot be determined from the response class, because they are highly dynamic and often just stored in a Map.
```

Comment:
Suggestion: In most cases, links cannot be determined from the response class, ...